fmt/src/helpers.rs

```diff
@@ -0,0 +1,15 @@
+use solang_parser::pt::*;
+
+pub fn namespace_matches(left: &Expression, right: &Expression) -> bool {
+    if matches!(left, Expression::MemberAccess(..)) || matches!(left, Expression::MemberAccess(..))
+    {
+        // TODO handle me
+        todo!()
```

Comment:
what needs to go here?

Reply:
i'll expand the todo message here and return false. it shouldn't panic either way